dev-notes/metadata.md

```diff
@@ -0,0 +1,67 @@
+# Platform metadata
+
+Platform metadata contains information that uniquely identify the platform being
+attested. Metadata is composed of
+
+- Device manufacturer and model name
+- MAC address
+- Serial number
+- EK certificate hash
```

Comment:
@arturkow2000 shouldn't this be removed? The EK cert hash in not a part of metadata?

Reply:
Yes, it should.
Fixed.